package-scripts.js

```diff
@@ -38,8 +46,22 @@ const truffleCommand = `node ${joinPath(pathArcJsRoot, truffleIsInternal ? "node
 module.exports = {
   scripts: {
     lint: {
-      default: "eslint .",
-      fix: 'nps "lint --fix ."'
+      default: series(
+        "nps lint.ts",
+        "nps lint.js"
+      ),
+      ts: {
+        default: "tslint --project .",
+        andFix: "nps \"lint.ts --fix\""
+      },
+      js: {
+        default: "eslint ./test",
```

Comment:
should it be "eslint ." so eslint will run on all js files (e.g migrations,rootdir js files..)

Reply:
Done

Reply:
where ?

Reply:
Sorry, looks like I made the change in the wrong branch (the one I've moved on to).  Hang on.

Reply:
OK, now check, [here](https://github.com/daostack/arc.js/pull/97/commits/c42195a453eb944b9697e3f28a5d4bca80e8ae68)